tests/test_scylladb.c

```diff
@@ -5,10 +5,12 @@
  * terms of the MIT license. A copy of the license can be found in the file
  * "LICENSE" at the root of this distribution.
  */
-
+#ifdef DB_ENABLE
```

Comment:
I wonder that are we able to set the `srcs` of bazel rule into an empty file or just not build it in bazel instead of using define guard in source file

Reply:
I sets the `srcs` to another empty test.